input/size.go

```diff
@@ -213,7 +213,7 @@ const (
 	//      - witness_script (to_local_script)
 	ToLocalPenaltyWitnessSize = 1 + 1 + 73 + 1 + 1 + ToLocalScriptSize
 
-	// AcceptedHtlcScriptSize 139 bytes
+	// AcceptedHtlcScriptSize 142 bytes
```

Comment:
Don't we need an entirely new set of constants for anchors? Or turn all these constants into functions so they can always return the correct size estimate based on the commitment type. 

Or are we OK with just always over estimating the _weight_ of the commitment? If we always over estimate (across the board), but the other implementations don't (in favor of more accurate estimates), then seems we could de-synchronize.

Reply:
It is only used from the `SizeUpperBound` method, so the decision to slightly overestimate in some cases was made. Alternative would be, as you say, to make a new set of constants, or make it a method, but this would also require us to create a new `WitnessType` for these confirmed types, which feels a bit overkill.

Ultimately we should probably rewrite weight estimation to rely less on constants, and instead inspect the actual transaction being created.

> If we always over estimate (across the board), but the other implementations don't (in favor of more accurate estimates), then seems we could de-synchronize.

These constants are only being used for weight estimation, so we should be okay here.